docs/OCP_CI_Tutorials/Cluster_Cleanup/Manual_Cluster_Cleanup_Guide.md

```diff
@@ -4,13 +4,45 @@
 
 - [Introduction](#introduction)
 - [AWS](#aws)
+  - [General cleanup dos and don'ts](#general-cleanup-dos-and-donts)
+    - [VPCs](#vpcs)
+    - [Hosted zones](#route-53-hosted-zones)
+    - [S3 Buckets](#s3-buckets)
+  - [Cleanup resources by cluster identified](#cleanup-resources-by-cluster-identified)
 
 ## Introduction
 
 Sometimes, a scenario can fail before a cluster is fully deprovisioned leaving stale resources in a cloud-provider (at the time of writing this, we only use AWS). If that occurs, the Interop team is responsible for cleaning that cluster up in the cloud-provider account to avoid any unwanted cost. This document will serve as a guide to how to manually cleanup a cluster in the cloud-provider platforms we use.
 
 ## AWS
 
```

Comment:
## Prerequisites

In order to perform manual cleanup and be able to delete resources in the cloud-provider account, users must belong to the *watchers* group, which subscribes to the  _EC2-delete_  policy and necessary permissions.

Reply:
@oharan2 - We should confirm what group and minimum permissions are required to perform these actions.

Reply:
Agree, suggestion added. Lets take the group issue to Slack.